reference/observability/fields-and-object-schemas/logs-app-fields.md

```diff
@@ -3,9 +3,10 @@ mapped_pages:
   - https://www.elastic.co/guide/en/observability/current/logs-app-fields.html
 ---
 
-# Logs Explorer fields [logs-app-fields]
+# Discover logs fields [logs-app-fields]
```

Comment:
Same as the comment above https://github.com/elastic/docs-content/pull/679/files#r2028585586